pkg/controller/rbac/namespace/reconciler.go

```diff
@@ -118,10 +118,11 @@ func WithRoleRenderer(rr RoleRenderer) ReconcilerOption {
 // NewReconciler returns a Reconciler of Namespaces.
 func NewReconciler(mgr manager.Manager, opts ...ReconcilerOption) *Reconciler {
 	r := &Reconciler{
-		// TODO(negz): Is Updating appropriate here? Probably.
+		// NOTE(hasheddan): Patching applicator is used to avoid conflict with
+		// Kubernetes RBAC aggregation controller.
 		client: resource.ClientApplicator{
 			Client:     mgr.GetClient(),
-			Applicator: resource.NewAPIUpdatingApplicator(mgr.GetClient()),
+			Applicator: resource.NewAPIPatchingApplicator(mgr.GetClient()),
```

Comment:
I think we may actually be running into issues because we are both "owning" and "watching" cluster roles. I am going to remove the "owning" and test again.

Reply:
Okay so I have improved the situation by having the `ProviderRevision` RBAC controllers not own `ClusterRole` and `ClusterRoleBinding` respectively. This addresses the issue of continuous updates as it appears the namespace and providerrevision RBAC controllers were triggering each other repetitively. However, we are still "boiling the ocean" every 60 seconds due to https://github.com/crossplane/crossplane/issues/1923. I'll try to address that now.